testapi.pm

```diff
@@ -536,7 +541,14 @@ returns the base URL to contact the local os-autoinst service
 
 sub autoinst_url() {
     # move to backend?
-    return "http://10.0.2.2:" . (get_var("QEMUPORT") + 1);
+    if (get_var("CONNECT_PASSWORD")) {    #CONNECT_PASSWORD is used in HACLUSTER
+        use Net::Domain;
+        my $fqdn = Net::Domain::hostfqdn();
```

Comment:
Net::Domain was first released with perl v5.7.3 - that's good enough not to put in DEPENDENCIES :)

I wonder if it shouldn't be possible to always use fqdn instead of the rather unknown 10.0.2.2